### ISSUE TITLE
List Block: Cursor jumps to end of block after each keystroke

### ISSUE THREAD

**Issue by tritos-design** [getkirby/kirby#3168]
**Describe the bug**  
In list blocks in the panel blocks field the cursor jumps to the end of the block after each typed character.
Additionally, a "space" is not accepted as the last character in the block, which makes writing separate words impossible. You can add space in the middle of the text, though (with the jumping cursor issue as described).

**To Reproduce**  
Add a blocks field to a page with the list block enabled.
Open the page in the panel, add a list block and write some text in the block.

**Expected behavior**  
Cursor does not jump to the end of the block and white space at the end of the block is accepted.

**Kirby Version**  
3.5.3 (currently latest)

**Console output**  
None

**Desktop (please complete the following information):**  
 - OS: Linux Mint 20.1
 - Browser: Firefox
 - Version: 85.0.1

**Comment by tritos-design** [comment]
Just checked older versions of Kirby and it's working OK until 3.5.1.
The error is present in 3.5.2 and 3.5.3.

**Comment by afbora** [comment]
Related PR is https://github.com/getkirby/kirby/pull/3127

I guess it is an issue because it converts into paragraphs in every stroke. The issue seems to be resolved if we do this just while saving. What do you think? @bastianallgeier 

````php
'save' => function ($value): string {
  return empty($value) === false ? str_replace(['<p>', '</p>'], '', $value) : $value;
},
````

**Comment by bastianallgeier** [comment]
@afbora we need to double-check that but it seems like a better solution indeed.

**Comment by afbora** [comment]
It works correctly with `save` solution, but there is a general issue with the writer field. Since it always converts data, `savebar` appears.
If we fix with clearing on `save` method, #3126 issue will occur for list field. 
So sorry but I guess we need to revert this change.
We have to wait for `ProseMirror` to fix it.

**Comment by bastianallgeier** [comment]
✅
